app/src/services/chefs.ts

```diff
@@ -50,19 +50,32 @@ const service = {
         const response = (await chefsAxios(formId).get(`submissions/${submissionId}`)).data;
         const status = (await chefsAxios(formId).get(`submissions/${submissionId}/status`)).data;
 
+        const submission = response.submission.submission.data;
+
+        const financiallySupportedValues = {
+          financiallySupportedBC: isTruthy(submission.isBCHousingSupported),
+          financiallySupportedIndigenous: isTruthy(submission.isIndigenousHousingProviderSupported),
+          financiallySupportedNonProfit: isTruthy(submission.isNonProfitSupported),
+          financiallySupportedHousingCoop: isTruthy(submission.isHousingCooperativeSupported)
+        };
+
         await prisma.submission.create({
           data: {
             submissionId: response.submission.id,
             confirmationId: response.submission.confirmationId,
-            contactEmail: response.submission.submission.data.contactEmail,
-            contactPhoneNumber: response.submission.submission.data.contactPhoneNumber,
-            contactFirstName: response.submission.submission.data.contactFirstName,
-            contactLastName: response.submission.submission.data.contactLastName,
+            contactEmail: submission.contactEmail,
+            contactPhoneNumber: submission.contactPhoneNumber,
+            contactName: `${submission.contactFirstName} ${submission.contactLastName}`,
+            financiallySupported: Object.values(financiallySupportedValues).includes(true),
+            ...financiallySupportedValues,
             intakeStatus: status[0].code,
-            projectName: response.submission.submission.data.projectName,
-            queuePriority: parseInt(response.submission.submission.data.queuePriority),
-            singleFamilyUnits: response.submission.submission.data.singleFamilyUnits,
-            streetAddress: response.submission.submission.data.streetAddress,
+            latitude: parseInt(submission.latitude),
+            longitude: parseInt(submission.longitude),
+            naturalDisaster: submission.naturalDisasterInd,
+            projectName: submission.companyNameRegistered,
+            queuePriority: parseInt(submission.queuePriority),
+            singleFamilyUnits: submission.singleFamilyUnits ?? submission.multiFamilyUnits,
```

Comment:
Will this safely handle situations where singleFamilyUnits is an empty string with whitespace? "  " would end up getting injected, which I suspect would not be semantically useful or even detrimental.